main.tf

```diff
@@ -1,48 +1,38 @@
 provider "aws" {
-  alias  = "acm_provider"
   region = "us-east-1"
+  alias  = "acm_provider"
 }
 
 data "aws_route53_zone" "selected" {
   name = "${var.parent_zone_name}"
 }
 
 module "aws-cert" {
-  source = "github.com/lean-delivery/tf-module-aws-acm"
-
-  providers = {
-    aws = "aws.acm_provider"
-  }
+  source  = "terraform-aws-modules/acm/aws"
+  version = "~> v2.0"
```

Comment:
Is it upgrade to 0.12?
If not - please check module docs:
https://github.com/terraform-aws-modules/terraform-aws-acm#terraform-versions

If yes - please update syntax according 0.12 specification.

Reply:
From module readme - Terraform 0.12. Pin module version to ~> v2.0